tests/New-DbaLogin.Tests.ps1

```diff
@@ -48,18 +48,20 @@ Describe "$CommandName Integration Tests" -Tags "IntegrationTests" {
         }
     } catch { <#nbd#> }
 
-    #create Windows login
-    $computer = [ADSI]"WinNT://$computerName"
-    try {
-        $user = [ADSI]"WinNT://$computerName/$credLogin,user"
-        if ($user.Name -eq $credLogin) {
-            $computer.Delete('User', $credLogin)
-        }
-    } catch { <#User does not exist#> }
+    if ($IsWindows -ne $false) {
+        #create Windows login
+        $computer = [ADSI]"WinNT://$computerName"
+        try {
+            $user = [ADSI]"WinNT://$computerName/$credLogin,user"
+            if ($user.Name -eq $credLogin) {
+                $computer.Delete('User', $credLogin)
+            }
+        } catch { <#User does not exist#> }
 
-    $user = $computer.Create("user", $credLogin)
-    $user.SetPassword($password)
-    $user.SetInfo()
+        $user = $computer.Create("user", $credLogin)
+        $user.SetPassword($password)
+        $user.SetInfo()
```

Comment:
This fails to run on my Windows systems. Stepping through it as well and `Create()` is not a valid method for Windows 10 machine. If we really need to test Windows login we can always mock it up.

Reply:
I'm not sure why it fails on your machine, cause this particular test was written 2+ years ago and ever since happily ran on Appveyor without any issues.

Reply:
Yeah I'm not catching why that method is not available. I'm on 2004 build of Windows 10 but don't have an older version to test on..

But intial issue would be it is trying to create a local account on my remote SQL Server instance... The command in the test is run locally, not against the remote machine.

Reply:
iirc, it would work with both remote and local setups, as back in the day, I was testing it against a remote server and it worked.
In any case, this PR is not changing the way tests work, it only makes Windows-related section conditional for the test to run on a non-Windows platform. If that is a concern I can revert that part.